chromium_src/components/sync/android/sync_service_android_bridge.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_SYNC_ANDROID_SYNC_SERVICE_ANDROID_BRIDGE_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_SYNC_ANDROID_SYNC_SERVICE_ANDROID_BRIDGE_H_
+
+// We need to expand SyncUserSettings with
```

Comment:
This should be in chromium_src/components/sync/service/sync_user_settings.h

Reply:
Sorry, indeed, fixed at 
https://github.com/brave/brave-core/compare/dacc39401ba06ecc40509cffa6349a84601d1caf..d74ed1ae25a1700f0e74612f28d156ec41c9b390